flash/core/data/utilities/paths.py

```diff
@@ -133,6 +133,18 @@ def list_valid_files(
     return [path for path in paths if has_file_allowed_extension(path, valid_extensions)]
 
 
+def _show_filter_valid_files_message(
+    invalid_extensions: Optional[Tuple[str, ...]] = None, valid_extensions: Optional[Tuple[str, ...]] = None
+) -> None:
+    invalid_extensions_message = ", ".join(invalid_extensions)
+    valid_extensions_message = ", ".join(valid_extensions)
+    message = (
+        f"Found invalid file extensions: {invalid_extensions_message}\n"
+        + f"The supported file extensions are: {valid_extensions_message}"
+    )
+    print(message)
```

Comment:
Would it be better if we raise a warning here?

```suggestion
    raise UserWarning(message)
```

Warnings usually come with extra benefits, including the ability to ignore these messages, in case someone wants a cleaner output on the console.

Reply:
the original issue only stated to create an error message; other PR directives are to make sure PRs only do 1 thing. Adding a flag to suppress warnings maybe warrants a new issue and PR after this first feature request is resolved.